packages/datagateway-common/src/homePage/homePage.component.tsx

```diff
@@ -58,122 +57,126 @@ const styles = (theme: Theme): StyleRules =>
     },
   });
 
-interface HomePageProps {
-  res: HomepageContents | undefined;
+export interface HomePageProps {
+  title: string;
+  howLabel: string;
+  exploreLabel: string;
+  exploreDescription: string;
+  discoverLabel: string;
+  discoverDescription: string;
+  downloadLabel: string;
+  downloadDescription: string;
   logo: string;
   backgroundImage: string;
   exploreImage: string;
   discoverImage: string;
   downloadImage: string;
+
+  [key: string]: string;
 }
 
-export type CombinedHomePageProps = HomePageProps & WithStyles<typeof styles>;
+type CombinedHomePageProps = HomePageProps & WithStyles<typeof styles>;
 
-class HomePage extends React.Component<CombinedHomePageProps> {
-  getString = (key: string): string => {
-    return this.props.res && this.props.res[key] ? this.props.res[key] : key;
+const HomePage = (props: CombinedHomePageProps): React.ReactElement => {
+  const getString = (key: string): string => {
+    return props[key] !== undefined ? props[key] : key;
```

Comment:
We don't actually need this function any more as the `t` function handles this logic so it will always pass a `string`. Plus all the types of the props are `string` not `string | undefined` so I think it would just be easier to refer to them directly.

Reply:
Ahh good spot, I had missed that